app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        _command, file, new_file = command.split()
```

Comment:
Consider using a more descriptive variable name instead of `_command` to improve code readability. According to the checklist, descriptive and correct variable names should be used.